packages/react-magma-dom/src/components/CharacterCounter/CharacterCounter.test.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { axe } from '../../../axe-helper';
+import { Input } from '../Input';
+import { CharacterCounter } from '.';
+import { transparentize } from 'polished';
+import { magma } from '../../theme/magma';
+import { render, fireEvent } from '@testing-library/react';
+import { I18nContext } from '../../i18n';
+import { defaultI18n } from '../../i18n/default';
+
+const testId = 'test-id';
```

Comment:
Since you use `const labelText = 'Character Counter';` in every test, you can also declare that up here instead of recreating the variable in each test.

Reply:
Same with `characterLeft` and `characterOver` since you're pulling that from i18n and won't change